components/datetime/src/helpers.rs

```diff
@@ -14,41 +14,39 @@
 ///
 /// The size should correspond to the Rust version in rust-toolchain.toml.
 ///
-/// If the size on latest nightly differs from rust-toolchain.toml, use the
+/// If the size on latest beta differs from rust-toolchain.toml, use the
 /// named arguments version of this macro to specify both sizes:
 ///
 /// ```ignore
-/// size_test!(MyType, my_type_size, pinned = 32, nightly = 24);
+/// size_test!(MyType, my_type_size, pinned = 32, beta = 24);
```

Comment:
I'd like these tests to run on latest nightly so that we can learn immediately about regressions or improvements.